Source/ObjectiveDropboxOfficial/Shared/Handwritten/Networking/DBTransportDefaultClient.m

```diff
@@ -42,18 +46,27 @@ - (instancetype)initWithAccessToken:(NSString *)accessToken
       if (transportConfig.sharedContainerIdentifier) {
         backgroundSessionConfig.sharedContainerIdentifier = transportConfig.sharedContainerIdentifier;
       }
+
+      NSOperationQueue *secondarySessionDelegateQueue = [[NSOperationQueue alloc] init];
+      secondarySessionDelegateQueue.maxConcurrentOperationCount = 1; // [Michael Fey, 2017-05-16] From the NSURLSession documentation: "The queue should be a serial queue, in order to ensure the correct ordering of callbacks."
+      secondarySessionDelegateQueue.name = [NSString stringWithFormat:@"%@ Secondary NSURLSession delegate queue", NSStringFromClass(self.class)];
+      secondarySessionDelegateQueue.qualityOfService = NSQualityOfServiceUtility;
```

Comment:
Repeating yourself here. It might make sense to method called `- (NSOperationQueue *)sessionQueueWithName:(NSString *)queueName` and move those four lines in there.